RealmSwift-swift1.2/Object.swift

```diff
@@ -191,6 +191,27 @@ public class Object: RLMObjectBase, Equatable, Printable {
         }
     }
 
+    // MARK: Dynamic list
+
+    /**
+    This method is useful only in specialized circumstances, for example, when building
```

Comment:
This comment is written in the Swift 2.0 documentation style. Please use `:warning:`, `:param: propertyName` and `:returns:`.